### ISSUE TITLE
Incompatible with jQuery 3.x

### ISSUE THREAD

**Issue by dieisraels** [worseisbetter/galleria#384]
Galleria 1.4.7 isn't compatible with jQuery 3:

> JQMIGRATE: jQuery.fn.load() is deprecated

at Galleria.loadTheme: 

`$( window ).load( function() { ... });`

**Comment by macu** [comment]
I got around this with an ugly hack, in the meantime, rather than using an older version of jQuery:

```JavaScript
$.fn.load = function() {
  var a = $.merge(["load"], arguments);
  return this.on.apply(this, a);
};
```

**Comment by aibon** [comment]
This will be fixed in the next version. Until then you could simply use jQuery migrate if you need jQuery3, please see [#373](https://github.com/worseisbetter/galleria/issues/373)

**Comment by dieisraels** [comment]
"dist" folder is not updated in release 1.5.0. Can you update this?

**Comment by aibon** [comment]
sorry, don't push releases in a hurry ;(

**Comment by dieisraels** [comment]
There are still problems with jQuery 3.x in the dist folder Release 1.5.1:

> JQMIGRATE: jQuery.fn.load() is deprecated at loadTheme

Can you check this again:
https://github.com/worseisbetter/galleria/tree/1.5.1

Because this is the version I get with bower update...

**Comment by aibon** [comment]
the tag got a wrong commit hash, just released 1.5.2
